app/services/data_migrations/backfill_application_choice_personal_statement.rb

```diff
@@ -0,0 +1,24 @@
+module DataMigrations
+  class BackfillApplicationChoicePersonalStatement
+    TIMESTAMP = 20230612121150
+    MANUAL_RUN = true
+
+    def change
+      ApplicationForm.select(:id).in_batches(of: 100) do |relation|
+        ids = relation.ids.join(',')
+
+        ActiveRecord::Base.connection.execute(<<~SQL)
+          UPDATE "application_choices" c
+          SET personal_statement = (
+            SELECT becoming_a_teacher
+            FROM application_forms
+            WHERE c.application_form_id = id
+            AND id in (#{ids})
+            AND becoming_a_teacher IS NOT NULL
+          )
+          where c.application_form_id in (#{ids})
+        SQL
+      end
+    end
```

Comment:
Good work! I'd change this to only get application forms which their application choices doesn't match the personal statement as more like a cautious approach like if this migration fails midway you don't need start from zero again (as we have more than 100k application forms starting from zero can be time consuming). Wdyt?